src/background/errors.js

```diff
@@ -32,6 +32,8 @@ const IGNORED_ERRORS = [
   // Attempts to access pages for which Chrome does not allow scripting
   /Cannot access contents of.*/,
   /The extensions gallery cannot be scripted/,
+  // The extension is blocked on LMSs
```

Comment:
We need to be clear that the extension is only disabled in Hypothesis LMS assignments to avoid confusion for students and teachers between the extension and the embedded Hypothesis instance. The user can still use the extension on other pages hosted in the LMS itself.